views/js/controller/CompatibilityChecker/diagnostics.js

```diff
@@ -40,19 +39,7 @@ define([
 
             showMessage('#feedback-box');
 
-            diagnosticFactory($contentArea, config)
-                .on('start', function() {
-                    loadingBar.start();
-                })
-                .on('end', function() {
-                    loadingBar.stop();
-                })
-                .on('render', function() {
-                    loadingBar.stop();
-                    if (config.autoStart) {
-                        this.run();
-                    }
-                });
+            diagnosticFactory($contentArea, config);
```

Comment:
Please revert this change, and follow the above comment (the autostart management must still be moved in the component however)